channels/ht.m3u

```diff
@@ -1,11 +1,9 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="French" tvg-logo="https://i.imgur.com/uisopuM.png" group-title="Music",Nago TV
 http://haititivi.com:8088/haititv/tele6NY/index.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="French" tvg-logo="https://i.imgur.com/uisopuM.png" group-title="Music",Nago TV
-http://haititivi.com:8088/haititv/tele6NY/tracks-v1a1/index.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="French" tvg-logo="https://i.imgur.com/HFyrVaS.png" group-title="",Radio TeleBoston (USA)
 http://162.244.81.145:3333/live//telebostonrtb/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="French" tvg-logo="https://i.imgur.com/5nYeBNN.jpg" group-title="",Tele Haiti
-http://66.175.238.147:1935/live/myStream/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="French" tvg-logo="https://static-media.streema.com/media/cache/17/02/1702c5903e8733db65b4902068424029.jpg" group-title="",Telehaiti (Not 24/7)
+http://66.175.238.147:1935/live/myStream/playlist.m3u8
```

Comment:
Why did you replace the correct logo and name with the wrong one?

Reply:
Ah ! Well... x) i'm guilty for this one !